recipes/constava/meta.yaml

```
@@ -0,0 +1,118 @@
+    conformational ensemble, the phi- and psi-angles for each conformational state in the ensemble
+    need to be provided.
+
+    As input data Constava needs the backbone dihedral angles extracted from the conformational ensemble.
+    These dihedrals can be obtained using GROMACS' gmx chi module (set `--input-format=xvg`)
+    or using the constava dihedrals submodule, which supports a wide range of MD and structure formats.
+
+    The software is subdivided in three submodules:
+
+    The constava dihedrals submodule provides a simple way to extract backbone dihedral angles from MD
+    simulations or PDB ensembles. For more information run: `constava dihedrals -h`. Alternatively,
+    the backbone dihedrals may be extracted with GROMACS' gmx chi module.
+
+    The constava analyze submodule analyzes the provided backbone dihedral angles and infers
+    the propensities for each residue to reside in a given conformational state. For more information run:
+    `constava analyze -h`.
+
+    The constava fit-model can be used to train a custom probabilistic model of conformational states. The
+    default models were derived from an analysis of NMR ensembles and chemical shifts; they cover six
+    conformational states:
+
+    1. Core Helix - Exclusively alpha-helical, low backbone dynamics
+    2. Surrounding Helix - Mostly alpha-helical, high backbone dynamics
+    3. Core Sheet - Exclusively beta-sheet, low backbone dynamics
+    4. Surrounding Sheet - Mostly extended conformation, high backbone dynamics
+    5. Turn - Mostly turn, high backbone dynamics
+    6. Other - Mostly coil, high backbone dynamics
+
+    For more information run: `constava fit-model -h`.
+
+    This software tool has been developed by the Bio2Byte research group at Vrije Universiteit Brussel (VUB)
+    in Belgium and it is distributed under the GNU General Public License v3 (GPLv3) License.
+
+    Please cite:
+
+    Jose Gavalda-Garcia, David Bickel, Joel Roca-Martinez, Daniele Raimondi, Gabriele Orlando, Wim Vranken,
+    Data-driven probabilistic definition of the low energy conformational states of protein residues,
+    NAR Genomics and Bioinformatics, Volume 6, Issue 3, September 2024, lqae082,
+    https://doi.org/10.1093/nargab/lqae082 (Published: 09 July 2024)
+
+    Contact us: <bio2byte@vub.be> or <wim.vranken@vub.be>
+  license: GPLv3
+  license_file: LICENSE
+  dev_url: https://github.com/bio2byte/constava
+  doc_url: https://pypi.org/project/constava/
+  doc_source_url: https://github.com/Bio2Byte/constava/blob/main/README.md
+
+extra:
+  author: Wim Vranken <wim.vranken@vub.be>, Jose Gavalda-Garcia <jose.gavalda.garcia@vub.be>, David Bickel <david.bickel@vub.be>, Joel Roca-Martinez <joel.roca.martinez@vub.be>, Daniele Raimondi <daniele.raimondi@kuleuven.be>, Gabriele Orlando <gabriele.orlando@kuleuven.be>
+  recipe-maintainers:
+    - Adrián Díaz <adrian.diaz@vub.be> (@agdiaz)
```

I believe those should be just the Github ids
```suggestion
    - @agdiaz
```

---

Good catch, thanks. I'm on it!